packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -269,18 +269,18 @@ Generated by [AVA](https://ava.li).
       @netlify/build 1.0.0␊
     ␊
     > Flags␊
-      config: /file/path␊
+      config: packages/build/tests/core/fixtures/empty/netlify.toml␊
       debug: true␊
       featureFlags: {}␊
       testOpts:␊
         pluginsListUrl: test␊
         silentLingeringProcesses: true␊
     ␊
     > Current directory␊
-      /file/path␊
```

Comment:
Wondering if the missing current directory is expected for this fixture? 🤔

Reply:
This is because we're running this fixture from cwd, so it becomes an empty string when it gets transformed to a relative path. Do you think we should be handling this case specifically and transforming it to something like `/cwd/path`?

Reply:
Ah, you're right! I guess because this is an empty config file our `cwd` isn't changed as it happens for the other tests in this snapshot?

Reply:
Exactly.